q2cli/handlers.py

```diff
@@ -336,12 +336,12 @@ def get_click_options(self):
             name = '--' + self.cli_name
 
         if self.default is NoDefault:
-            yield click.Option([name], type=type, help='[required]')
```

Comment:
Sorry! I misread the code, and thought we were in the `if type is bool` block here.